addons/website_event_track/models/website.py

```diff
@@ -14,8 +13,18 @@
 class Website(models.Model):
     _inherit = "website"
 
-    app_icon = fields.Image(string='Website App Icon', compute='_compute_app_icon', store=True, readonly=True, help='This field holds the image used as mobile app icon on the website (PNG format).')
-    events_app_name = fields.Char(string='Events App Name', compute='_compute_events_app_name', store=True, readonly=False, help="This fields holds the Event's Progressive Web App name.")
+    app_icon = fields.Image(
```

Comment:
Unnecessary diff, creates issues and is takes too much space for nothing( moreover we cannot see what changed, is there any difference ?)